src/main/java/uk/gov/hmcts/reform/sendletter/ResponseExceptionHandler.java

```diff
@@ -59,7 +61,7 @@ protected void activateDirectFieldAccess(DataBinder dataBinder) {
 
     @ExceptionHandler(InvalidTokenException.class)
     protected ResponseEntity<Void> handleInvalidTokenException(InvalidTokenException exc) {
-        log.warn(exc.getMessage(), exc);
+        log.warn(exc.getMessage(), new UnauthenticatedException(exc));
```

Comment:
can we make `InvalidTokenException` extend the required base class instead?

Reply:
Comes from library

Reply:
let's change it :)

Reply:
🎉